src/pages/RadicalCollaboration/Share/styles.js

```diff
@@ -0,0 +1,116 @@
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+const useStyles = makeStyles((theme) => ({
+  containerStyle: {
+    paddingTop: '96px',
+  },
+  imageStyle: {
+    border: '1px solid',
+    borderColor: theme.palette.background.darkGray,
+    maxWidth: '100%',
+    [theme.breakpoints.down('md')]: {
+      fontSize: '36px',
+    },
+  },
+  ctiImageStyle: {
+    maxWidth: '100%',
+    [theme.breakpoints.down('xs')]: {
+      width: '102px',
+      height: '62px',
+    },
+  },
+  ctiSquareStyle: {
+    maxWidth: '100%',
+    [theme.breakpoints.down('xs')]: {
+      width: '150px',
+      height: '150px',
+    },
+  },
+  starMediaSection: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    [theme.breakpoints.down('xs')]: {
+      flexDirection: 'column',
+    },
+  },
+  socialMediaSectionStyle: {
+    padding: '96px 0 96px',
+  },
+  mediaInfoStyle: {
+    padding: '96px 0 96px',
+  },
+  card: {
+    backgroundColor: '#FFFFFF',
+    height: '186px',
+    border: '1px solid',
+    borderRadius: '4px',
+    borderColor: theme.palette.background.darkGray,
```

Comment:
Same comment if you choose to condense
```suggestion
    border: `1px solid ${theme.palette.background.darkGray}`,
    borderRadius: '4px',
```